pkg/controller/sinks/sinks_test.go

```diff
@@ -63,7 +64,15 @@ func TestGetSinkURI(t *testing.T) {
 		wantErr   error
 		ref       *corev1.ObjectReference
 	}{
-		"happy": {
+		"happy - hostname": {
```

Comment:
I like the addition of this test ensuring we can still target v1alpha1 Addressables.